cmd/ceremony/cert_test.go

```diff
@@ -306,6 +306,21 @@ func TestVerifyProfile(t *testing.T) {
 			certType:    intermediateCert,
 			expectedErr: "issuer-url is required for intermediates",
 		},
```

Comment:
Here's an extra unit test you could add to `TestParseOID`. In `parseOID` there is a log line that states OID components should be `>= 1`.
```
iff --git a/cmd/ceremony/cert_test.go b/cmd/ceremony/cert_test.go
index cf67976a8..2e3dba8e8 100644
--- a/cmd/ceremony/cert_test.go
+++ b/cmd/ceremony/cert_test.go
@@ -38,7 +38,10 @@ func TestParseOID(t *testing.T) {
        test.AssertError(t, err, "parseOID accepted an empty OID")
        _, err = parseOID("a.b.c")
        test.AssertError(t, err, "parseOID accepted an OID containing non-ints")
-       oid, err := parseOID("1.2.3")
+       oid, err := parseOID("1")
+       test.AssertNotError(t, err, "parseOID failed with a valid OID")
+       test.Assert(t, oid.Equal(asn1.ObjectIdentifier{1}), "parseOID returned incorrect OID")
+       oid, err = parseOID("1.2.3")
```

Reply:
Ohhh I understand now. Good catch on the updated test you added.